lib/ansible/module_utils/iosxr.py

```diff
@@ -26,12 +26,34 @@
 # LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
-from ansible.module_utils._text import to_text
-from ansible.module_utils.basic import env_fallback, return_values
+
+import json
+import sys
+from difflib import Differ
+from six import StringIO
+from io import BytesIO
+from lxml import etree
```

Comment:
Please handle import error for third-party library import. Something similar to
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/junos.py#L31